python/tank_vendor/shotgun_authentication/session_cache.py

```diff
@@ -13,12 +13,13 @@
 in the site's cache location.
 """
 
+from __future__ import with_statement
 import os
 import sys
 import urlparse
-from ConfigParser import SafeConfigParser
 from tank_vendor.shotgun_api3 import Shotgun, AuthenticationFault, ProtocolError
 from tank_vendor.shotgun_api3.lib import httplib2
+from tank_vendor import yaml
```

Comment:
Can I chime in with Rob's feedback that he sent to you via email here - I wonder if this should be a class instead.